xinference/core/worker.py

```diff
@@ -694,10 +706,13 @@ async def launch_builtin_model(
 
     @log_async(logger=logger)
     async def terminate_model(self, model_uid: str):
-        event_model_uid, _, __ = parse_replica_model_uid(model_uid)
+        # Terminate model while its launching is not allow
+        if model_uid in self._model_uid_launching_guard:
+            raise ValueError(f"{model_uid} is launching")
+        origin_uid, _, __ = parse_replica_model_uid(model_uid)
```

Comment:
Why changed the name of the variable?

Reply:
there were duplicated calls, just unify the name
```
event_model_uid, _, __ = parse_replica_model_uid(model_uid)
...
origin_uid, _, __ = parse_replica_model_uid(model_uid)
 ```